app/gui2/src/composables/navigator.ts

```diff
@@ -4,7 +4,7 @@ import { useApproach } from '@/composables/animation'
 import { PointerButtonMask, useEvent, usePointer, useResizeObserver } from '@/composables/events'
 import { Rect } from '@/util/data/rect'
 import { Vec2 } from '@/util/data/vec2'
-import { computed, proxyRefs, ref, type Ref } from 'vue'
+import { computed, proxyRefs, ref as shallowRef, type Ref } from 'vue'
```

Comment:
Is `ref as shallowRef` intentional? It seems misleading

Reply:
Absolutely not, good eye. This must have been some kind of auto-import or refactor tool mishap.